src/LibraryManagementSystem.java

```diff
@@ -11,4 +11,39 @@ public class LibraryManagementSystem {
     public static void main(String[] args) {
 
     }
+    
+    public static void addBook (String tittle, String author, String ISBN, String quantity) {
+    	boolean isExist = false;
```

Comment:
please create a new method `getBookIndexByISBN(String ISBN)` it should return `integer`
- default value should be `-1`